services/frontend/process_configs.go

```diff
@@ -114,7 +114,8 @@ func (s *server) processConfigs() error {
 		s.deletingConfigs.Store(config.Id, nil)
 
 		go func(config *pb.Config) {
-			log.Info().Msgf("Deleting config %v", config.Id)
+			log.Info().Msgf("Deleting config: %v", config.Name)
+			log.Debug().Msgf("Deleting config: %s with id: %v", config.Name, config.Id)
```

Comment:
I don't think the debug-level message adds more context, compared to the info-level.